tests/integration/conftest.py

```diff
@@ -41,3 +48,42 @@ def app_client(app):
     """
     with app.test_client() as client:
         yield client
+
+
+class Mock_RNN_Learner(RNN_Learner):
```

Comment:
can you add a docstring to explain what you do here and why?

Reply:
Added